tests/dependency/poetry_installable_library/pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [tool.poetry]
-authors = ["Louis Hulot <louis.hulot@owkin.com>"]
+authors = ["Substra dev team"]
```

Comment:
Should we add the squad email there ?

Reply:
This is just a quick lib implementation to test the "local dependency feature". Not destinate to users at all.